src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+import duke.DukeException;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+/**
+ * Command to Mark or Unmark task in the taskList.
+ */
+public class MarkCommand extends Command {
+	private final boolean toMark;
```

Comment:
Maybe you can change the boolean's name to something more intuitive? (e.g: isMarked)

Reply:
Thank you for the suggestion 😄